patches/common/skia/fix_font_thickness.patch

```diff
@@ -0,0 +1,168 @@
+diff --git a/src/ports/SkFontHost_mac.cpp b/src/ports/SkFontHost_mac.cpp
+index 3cc51d43f6..c0b606c004 100644
+--- a/src/ports/SkFontHost_mac.cpp
++++ b/src/ports/SkFontHost_mac.cpp
+@@ -377,17 +377,28 @@ static constexpr const uint8_t kSpiderSymbol_ttf[] = {
+     0x00, 0x02, 0x00, 0x00
+ };
+ 
++enum class SmoothBehavior {
++    none, // SmoothFonts produces no effect.
++    some, // SmoothFonts produces some effect, but not subpixel coverage.
++    subpixel, // SmoothFonts produces some effect and provides subpixel coverage.
++};
++
+ /**
+  * There does not appear to be a publicly accessable API for determining if lcd
+  * font smoothing will be applied if we request it. The main issue is that if
+  * smoothing is applied a gamma of 2.0 will be used, if not a gamma of 1.0.
+  */
+-static bool supports_LCD() {
+-    static bool gSupportsLCD = []{
+-        uint32_t bitmap[16][16] = {};
++static SmoothBehavior smooth_behavior() {
++    static SmoothBehavior gSmoothBehavior = []{
++        uint32_t noSmoothBitmap[16][16] = {};
++        uint32_t smoothBitmap[16][16] = {};
+         UniqueCFRef<CGColorSpaceRef> colorspace(CGColorSpaceCreateDeviceRGB());
+-        UniqueCFRef<CGContextRef> cgContext(
+-                CGBitmapContextCreate(&bitmap, 16, 16, 8, 16*4, colorspace.get(), BITMAP_INFO_RGB));
++        SkUniqueCFRef<CGContextRef> noSmoothContext(
```

Comment:
`SkUniqueCFRef` was added recently in https://skia-review.googlesource.com/c/skia/+/155612, the skia version on our master doesn't have it, it should be `UniqueCFRef` in this patch.